### PR TITLE
Rtc 314 change encodings

### PR DESCRIPTION
**Pull request by roznawsk**

### CODE REVIEW

File: webrtc/src/benchmarkRtc.ts [hunk 1]
```diff
@@ -139,19 +140,27 @@ const cleanup = async (client: Client, browsers: Array<Browser>) => {
...
   const maxPeersInRoom = Math.min(args.peers, args.peersPerRoom);
   const fullRooms = Math.floor(args.peers / maxPeersInRoom);
   const peersInLastRoom = args.peers % args.peersPerRoom;
   const activePeersInLastRoom = Math.min(args.activePeers, peersInLastRoom);
 
-  const incomingTracks = fullRooms * args.activePeers + activePeersInLastRoom;
+  const incoming_tracks = fullRooms * args.activePeers + activePeersInLastRoom;
 
-  const outgoingTracks =
+  const outgoing_tracks =
```
Comment (Rados13) [inline webrtc/src/benchmarkRtc.ts:151]: Why did you change names from camelCase to snake_case?

File: webrtc/src/benchmarkRtc.ts [hunk 2]
```diff
@@ -139,19 +140,27 @@ const cleanup = async (client: Client, browsers: Array<Browser>) => {
...
     fullRooms * args.activePeers * (maxPeersInRoom - 1) +
     activePeersInLastRoom * (peersInLastRoom - 1);
 
-  return { incomingTracks, outgoingTracks };
+  const outgoingBandwidth =
+    encodingBitrate.get(args.targetEncoding)! * incoming_tracks;
```
Comment (Rados13) [inline webrtc/src/benchmarkRtc.ts:156]: Shouldn't we use `outgoingTracks` here?

File: webrtc/src/benchmarkRtc.ts [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
-const ENCODING_REPORT_PERIOD = 5;
-const INBOUD_TRACK_BANDWIDTH = 0.15 + 0.5 + 1.5;
-const OUTBOUND_TRACK_BANDWIDTH = 1.5;
+
+const encodingReportPeriod = 5;
+const encodingBitrate = new Map<TrackEncoding, number>([
+  ["l", 0.15],
+  ["m", 0.5],
+  ["h", 1.5],
+]);
```
Comment (Rados13) [inline webrtc/src/benchmarkRtc.ts:16]: Why did you change the naming convention for these variables?

File: webrtc/src/benchmarkRtc.ts [hunk 4]
```diff
@@ -139,19 +140,27 @@ const cleanup = async (client: Client, browsers: Array<Browser>) => {
...
     activePeersInLastRoom * (peersInLastRoom - 1);
 
-  return { incomingTracks, outgoingTracks };
+  const outgoingBandwidth =
+    encodingBitrate.get(args.targetEncoding)! * incoming_tracks;
+
+  let incomingBandwidth = 0;
+  for (const encoding of args.availableEncodings) {
+    incomingBandwidth += encodingBitrate.get(encoding)! * incoming_tracks;
+  }
```
Comment (Rados13) [inline webrtc/src/benchmarkRtc.ts:161]:
```suggestion
  let incomingBandwidth = args.availableEncodings.reduce( (acc, encoding) => acc + encodingBitrate.get(encoding)! * incoming_tracks, 0)
```

File: webrtc/src/benchmarkRtc.ts (hunk 3 above)
Comment (roznawsk) [reply webrtc/src/benchmarkRtc.ts:16]: Although there are many conventions for naming constants, I think I prefer the Camel Case, also other constants also follow this rule.

**Review by Rados13 — APPROVED** [review]
